src/libs/actions/IOU.js

```diff
@@ -75,6 +77,30 @@ function requestMoney(report, amount, currency, recipientEmail, participant, com
         iouReport = ReportUtils.buildOptimisticIOUReport(recipientEmail, debtorEmail, amount, chatReport.reportID, currency, preferredLocale);
```

Comment:
Unrelated to this PR but should not the amount here be in cents?

Reply:
I dont think this has changed and it worked before so probably not but it would be good to confirm